pyigm/metallicity/mcmc.py

```diff
@@ -101,12 +101,73 @@
 
 from linetools import utils as ltu
 
+
+##For logUconstraint
+import sys
+sys.path.append(os.getcwd())
```

Comment:
Not after I moved `UVB_HM05.dat` and `UVB_HM12.dat` into `pyigm/pyigm/data/euvb/` and then used `pkg_resources` to locate them.  Removed.